src/user/room.ts

```diff
@@ -428,23 +428,31 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = Array.isArray(mention)
+                                    ? [...mention]
+                                    : mention ? [mention] : []
+
+    const mentionAliasPromiseList = await Promise.all(
+                                      replyToList
+                                      .map(c => this.alias(c) || c.name()))
+
+    const mentionAliasList: string[] = mentionAliasPromiseList
```

Comment:
This line can be removed because the array should not include the null anymore because of the above code `this.alias(c) || c.name()`

Reply:
but it is possible promise fail and got null value for ```c.name()```, isn't it?

Reply:
Before rename: 
![image](https://user-images.githubusercontent.com/25014776/46334435-dd3a6680-c5d8-11e8-8fae-6f72f91d718d.png)

Reply:
`c.name()` will always return a `string`, so it should never be a `null`.

However, you can keep this code and I'll have a look after we merged it.

Reply:
ok.